webpack.config.js

```diff
@@ -6,6 +6,7 @@ var index = require('./lib/index-static.jsx');
 
 module.exports = {
   entry: './lib/main.jsx',
+  devtool: 'source-map',
```

Comment:
Whoa, yeah, you're totally right, setting `devtool` to `eval` *does* make console logging work on Firefox!!!!

![2015-03-05_12-26-17](https://cloud.githubusercontent.com/assets/124687/6510256/e2f28820-c332-11e4-808b-593ef0308c5d.jpg)

WOW.

Unfortunately, it's totally dumb that this won't work on production, since it uses `eval()` to evaluate the code and thus won't work w/ minified stuff. It really sucks that Firefox's support for `.map.` files is limited to the debugger only.

Another thing I've just noticed about the `eval` approach is that it's actually only showing us the JSX file *after* the [JSX transform](http://facebook.github.io/react/docs/jsx-in-depth.html#the-transform). Which for Firefox console logging is still way better than referring to `bundle.js`, but for all other use cases (e.g. Chrome, the FF debugger) it's *worse*, since those other use cases show us the original JSX source when we use `.map.` files. So there is no clear winner here, which is lame.